devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy

```diff
@@ -37,6 +37,8 @@ metadata {
 		
 		// EZEX
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0006", outClusters: "0006, 000A, 0019", model: "E220-KR1N0Z0-HA", deviceJoinName: "eZEX Switch" //EZEX Switch
+        fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0006", outClusters: "0006, 000A, 0019", model: "E220-KR1N0Z1-HA", deviceJoinName: "eZEX Switch" //EZEX Switch
+        fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0006", outClusters: "0006, 000A, 0019", model: "E220-KR1N0Z2-HA", deviceJoinName: "eZEX Switch" //EZEX Switch
```

Comment:
Please fix the indentations (change 4 spaces to 1 tab)

Reply:
Dear @tklee2020 , could you please fix the indentations by change 4 spaces to 1 tab ? Many thanks